### PR TITLE
Add extra small player view w/ only play button

### PR DESCRIPTION
**Pull request by luwes**
Adds an extra small view with just a play button icon.

SCR-20220217-s4a

### CODE REVIEW

**Review by cjpillsbury — COMMENTED** [review]
A little worried about loosing mute for this view. Gonna hold off on approval until we can chat with the team. Otherwise, 😎

**Comment by gkatsev** [comment]
Hiding the captions button is also a bit of an issue.
What about putting the mute button top left corner and captions button top right corner?

**Comment by luwes** [comment]
SCR-20220218-nhx
SCR-20220218-ni5
SCR-20220218-nih

File: packages/mux-player/src/template.ts [hunk 1]
```diff
@@ -195,6 +225,24 @@ export const VodChromeLarge = (props: MuxTemplateProps) => html`
   </media-control-bar>
 `;
 
+// prettier-ignore
+export const LiveChromeExtraSmall = (props: MuxTemplateProps) => html`
```
Comment (cjpillsbury) [inline packages/mux-player/src/template.ts:229]: nit (blocking): If these are identical (for now), let's share the template fn as:
`export const LiveChromeExtraSmall = VodChromeExtraSmall;`

File: packages/mux-player/src/template.ts [hunk 2]
```diff
@@ -152,6 +164,24 @@ const MediaFullscreenButton = () => html`
   ${icons.FullscreenExit()}
 </media-fullscreen-button>`;
 
+// prettier-ignore
+export const VodChromeExtraSmall = (props: MuxTemplateProps) => html`
```
Comment (cjpillsbury) [inline packages/mux-player/src/template.ts:168]: non-blocking: PIP/Fullscreen buttons weren't discussed, but I'm cool with adding them and iterating as this hasn't gone through design and is easy enough to update.

**Review by cjpillsbury — CHANGES_REQUESTED** [review]
One small ask, otherwise lgtm.

File: packages/mux-player/src/utils.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ export function stylePropsToString(props: any) {
   Object.entries(props).forEach(([key, value]) => {
     style += `${kebabCase(key)}: ${value}; `;
   });
-  return style.trim();
+  return style ? style.trim() : undefined;
```
Comment (cjpillsbury) [inline packages/mux-player/src/utils.ts:6]: non-blocking: any reason this is yielding `undefined`?

File: packages/mux-player/src/styles.css [hunk 4]
```diff
@@ -56,7 +62,7 @@ media-controller:is([media-paused], :not([user-inactive]))::part(vertical-layer)
   --media-control-background: transparent;
   --media-control-hover-background: transparent;
   padding: 0;
-  width: min(9%, 90px);
+  width: max(27px, min(9%, 90px));
```
Comment (cjpillsbury) [inline packages/mux-player/src/styles.css:65]: non-blocking: This feels brittle. Will this break if the breakpoint changes?

File: packages/mux-player/src/styles.css [hunk 5]
```diff
@@ -25,6 +25,12 @@ media-control-bar [role="switch"] {
   height: 44px;
 }
 
+.size-extra-small media-control-bar [role="button"],
```
Comment (cjpillsbury) [inline packages/mux-player/src/styles.css:28]: interesting solution to "categorize" buttons. Might break down over time, but so would any `:is()` list or similar. 👌

File: packages/mux-player/src/template.ts (hunk 2 above)
Comment (luwes) [reply packages/mux-player/src/template.ts:168]: yes seemed useful if people want to see the tiny video in a larger view

**Review by cjpillsbury — APPROVED** [review]

**Comment by dylanjha** [comment]
very nice 😎

**Pull request merged**